09-mapping.Rmd

```diff
@@ -337,12 +362,13 @@ Some useful layout settings are listed below (see Figure \@ref(fig:layout2) for
 source("code/09-layout2.R")
 ```
 
-The impact of changing the color settings listed above is illustrated in Figure \@ref(fig:layout3) (see `?tm_layout` for full list).
+The impact of changing the color settings listed above is illustrated in Figure \@ref(fig:layout3) (see `?tm_layout` for a full list).
 
 ```{r layout3, fig.cap="Illustration of selected color-related layout options.", echo=FALSE}
 source("code/09-layout3.R")
 ```
 
+<!-- Maybe reverse order and progress from the high- to the low-level control. Overall, this section reads a bit like a vignette. Is this the intention. -->
 Beyond the low-level control over layouts and colors, **tmap** also offers high-level styles, using `tm_style_` functions (representing the second meaning of 'style' in the package).
```

Comment:
Maybe reverse order and progress from the high- to the low-level control. Overall, this section reads a bit like a vignette. Is this the intention?